2025 Haunted House/src/main/java/frc/robot/subsystems/MeniMech.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class MeniMech extends SubsystemBase {
+  
+  private double handUpTime = 7;
+  private double handDownTime = 3;
+  private double headUpTime = 5;
+  private double headDownTime = 5;
+  private Timer headTimer;
+  private Timer handTimer;
+  private Solenoid handSolenoid;
+  private Solenoid headSolenoid;
+
+
+  public MeniMech(Solenoid headSolenoid, Solenoid handSolenoid) {
+    this.headSolenoid= headSolenoid;
+    this.handSolenoid = handSolenoid;
+    headTimer= new Timer();
+    headTimer.start();
+    handTimer = new Timer();
+    handTimer.start();
```

Comment:
Maybe consider creating all your timers before starting them, especially if you want to have them somewhat synchronized. Do you know how long creating the timer object takes?

Reply:
fixed in [this commit](https://github.com/grt192/2024-Haunted-House/commit/dd91e1b7b2d58078fcbce140b21c609f1dc647bf)

Reply:
Love it.